js/build.js

```
@@ -1,193 +1,463 @@
+            // Show all containers with the same ID
+            await showAllContainersWithId(containerId, data, parent);
           }
 
-          return false;
+          resolve(instance);
         } catch (error) {
-          return 'string';
+          console.error('Error in conditional container:', error);
+          resolve(instance); // Resolve anyway to prevent blocking UI
         }
+      });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)